hack/update-openapi.sh

```diff
@@ -14,29 +14,21 @@ verify="${VERIFY:-}"
 output_path="${OUTPUT_PATH:-openapi}"
 output_package="${SCRIPT_ROOT}/${output_path}"
 
-# enumerate group versions
-ALL_FQ_APIS=() # e.g. k8s.io/kubernetes/pkg/apis/apps k8s.io/api/apps/v1
-INT_FQ_APIS=() # e.g. k8s.io/kubernetes/pkg/apis/apps
-EXT_FQ_APIS=("k8s.io/apimachinery/pkg/apis/meta/v1") # e.g. k8s.io/api/apps/v1
```

Comment:
`k8s.io/apimachinery/pkg/apis/meta/v1` isn't needed in this list as it's already on line 32